src/pages/components/chrome.mdx

```diff
@@ -122,41 +124,41 @@ Chrome uses [26px sized icons](/design/icons/chrome-icons) made specifically for
 
 <PropSheet components={props.data.mdx.components} componentName="footer" />
 
-### FooterItem
+#### Footer.Item
 
-<Component components={props.data.mdx.components} componentName="footerItem" />
+<Component components={props.data.mdx.subcomponents} componentName="footer.item" />
 
-<PropSheet components={props.data.mdx.components} componentName="footerItem" />
+<PropSheet components={props.data.mdx.subcomponents} componentName="footer.item" />
 
 ### Header
 
 <Component components={props.data.mdx.components} componentName="header" />
 
 <PropSheet components={props.data.mdx.components} componentName="header" />
 
-### HeaderItem
+#### Header.Item
 
-<Component components={props.data.mdx.components} componentName="headerItem" />
+<Component components={props.data.mdx.subcomponents} componentName="header.item" />
 
-<PropSheet components={props.data.mdx.components} componentName="headerItem" />
+<PropSheet components={props.data.mdx.subcomponents} componentName="header.item" />
 
-### HeaderItemIcon
+#### Header.ItemIcon
 
-<Component components={props.data.mdx.components} componentName="headerItemIcon" />
+<Component components={props.data.mdx.subcomponents} componentName="header.itemicon" />
 
-<PropSheet components={props.data.mdx.components} componentName="headerItemIcon" />
+<PropSheet components={props.data.mdx.subcomponents} componentName="header.itemicon" />
 
-### HeaderItemText
+#### Header.ItemText
 
-<Component components={props.data.mdx.components} componentName="headerItemText" />
+<Component components={props.data.mdx.subcomponents} componentName="header.itemtext" />
```

Comment:
[supernit] even though the `componentName` is case insensitive, it would be consistent to keep the camelCase here. `header.itemIcon`, `header.itemText`, etc. Not critical for merge, though.